### ISSUE TITLE
How  apply to new datasets?

### ISSUE THREAD

**Issue by xfby2016** [jnhwkim/orthoad#1]
Excellent work！How to start  if I want to apply it to a new dataset ？
I noticed that the project  are deeply bound specific dataset and must have ground_truth?

**Comment by jnhwkim** [comment]
In `dataset.py`, you can add your own dataset following the convention of PyTorch library, and need to add some extra codes (look the top of the source file) for smooth loading from the `train.py`.

For the just inference (no training), you can replace the annotation mask with zero tensors to match our output format of the Dataset class.
